concent_api/core/message_handlers.py

```diff
@@ -1362,8 +1362,8 @@ def is_message_recieved_in_wrong_state(subtask_id, forbidden_states):
     ).exists()
 
 
-def are_ids_unique_in_subtask_results_accepted_list(subtask_results_accepted_list):
+def are_subtask_ids_unique_in_subtask_results_accepted_list(subtask_results_accepted_list):
```

Comment:
Personally, I would change this to sth more general, i.e.:
```python
def are_items_unique(items: list):
    return len(items) == len(set(items))
```
and then in usage:

```python
are_items_unique([subtask_results_accepted.subtask_id for subtask_results_accepted in subtask_results_accepted_list])
```
in this way function has one single responsibility, is general and can be reused ;)

Also, it would made unit-testing much easier ;)

Reply:
Done